cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -29,8 +29,10 @@
  {% set feature_gates = "--feature-gates=" + grains.feature_gates -%}
 {% endif -%}
 
+{% set throttles = "--iptables-sync-period=1m --iptables-min-sync-period=10s" -%}
```

Comment:
The scenario I was afraid about is that you are setting bursts=2.

So if we are creating a replication controller with 3 replicas, if endpoint controller will be fast enough that it will be adding pods to Endpoints object one by one, the first two will be added quickly, but for the last we will need to wait 10s more.

I'm not sure how big problem it is though, so maybe it fine to leave now. We can tune later.